nodes/PayloadValidator.js

```diff
@@ -0,0 +1,52 @@
+const clone = require('clone');
+
+const variablesToCheck = [
+  'logger.metadata.organization',
+  'payload.system.organization',
+  'event.event.organization',
+  'event.event.token.contents.organization'
+];
+
+module.exports = class PayloadValidator {
+  constructor(_before) {
+    const before = clone(_before);
+    const {
```

Comment:
if there any way this deconstruction could fail?

Reply:
Only if they've deleted the items in the flow. I can add some safety around it